### ISSUE TITLE
Support for Jupyterlab ^2.0.0?

### ISSUE THREAD

**Issue by constd** [manuzhang/jupyterlab_spark#4]
Would it be possible to enable support for Jupyterlab 2.x.x?

currently, I'm getting this:
```
An error occured.
ValueError: The extension "jupyterlab_spark" does not yet support the current version of JupyterLab.


Conflicting Dependencies:
JupyterLab              Extension      Package
>=2.1.0 <2.2.0          >=0.19.1 <0.20.0@jupyterlab/application
```

**Comment by manuzhang** [comment]
@constd Thanks for reporting. I will try upgrading to JupyterLab 2.x.x

**Comment by manuzhang** [comment]
@constd 
[jupyterlab_spark 0.3.0](https://github.com/manuzhang/jupyterlab_spark/releases/tag/v0.3.0) has been released with JupyterLab 2.0 support. Please check it out.
